packages/react-components/react-checkbox/package.json

```diff
@@ -39,6 +39,7 @@
     "@fluentui/react-tabster": "^9.6.5",
     "@fluentui/react-theme": "^9.1.7",
     "@fluentui/react-utilities": "^9.8.0",
+    "@fluentui/react-jsx-runtime": "9.0.0-alpha.1",
```

Comment:
_nit:_ Can you move this above `@fluentui/react-label` in line `37` to keep alphabetical order?